src/Microsoft.Net.Http.Headers/DateTimeFormatter.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using static System.Text.Encoding;
+
+namespace Microsoft.Net.Http.Headers
+{
+    internal static class DateTimeFormatter
```

Comment:
I was testing changing to foreach to use Buffer.memcpy, but the results where identical.

Ref memcpy: https://gist.github.com/hallatore/a10ffd0d69a508d09b74a4f720c0511a